change-notes/1.23/analysis-javascript.md

```diff
@@ -11,6 +11,8 @@
   - [rate-limiter-flexible](https://www.npmjs.com/package/rate-limiter-flexible)
 
 * The call graph has been improved to resolve method calls in more cases. This may produce more security alerts.
+* import.meta is now supported in TypeScript.
```

Comment:
With this change I believe we have complete support for 3.6 so I'd rather just mention that we support TypeScript 3.6 now.

Bonus nitpick: add linebreak before the bullet point